superset/connectors/sqla/models.py

```diff
@@ -140,8 +142,14 @@ def get_time_filter(self, start_dttm, end_dttm):
             l.append(col <= text(self.dttm_sql_literal(end_dttm, is_epoch_in_utc)))
         return and_(*l)
 
-    def get_timestamp_expression(self, time_grain):
-        """Getting the time component of the query"""
+    def get_timestamp_expression(self, time_grain: Optional[str]) \
```

Comment:
If `time_grain` is optional shouldn’t there be a default?

Reply:
Based on the documentation I have found, `Optional[str]` is merely shorthand for `Union[None, str]`. In this case the argument is not optional, but can be either `None` or a `str`.